### PR TITLE
Document lack of guarantees with respect to the contents of diagnostics bundles

### PR DESCRIPTION
**Pull request by janisz**
## Description
https://jira.mesosphere.com/browse/DCOS_OSS-5146

Docs Ticket https://jira.mesosphere.com/browse/DCOS_OSS-5219
## Urgency
- [ ] Blocke
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/1.12/monitoring/index.md [hunk 1]
```diff
@@ -92,9 +92,13 @@ You can sort system health by `systemd` unit. However, this search can bring up
...
 
+### Diagnostics bundle contents
+
+The contents of the generated bundle are not stable over time and any internal or third party bundle analysis tooling should be programmed very defensively in this regard.
+
 ## Troubleshooting
 
-If you have any problems, you can check if the diagnostics service is running by SSHing to the Mesos leading master and checking the `systemd` status of the diagnostics component (`dcos-d3t.service`).
+If you have any problems, you can check if the diagnostics service is running by SSHing to the Mesos leading master and checking the `systemd` status of the diagnostics component (`dcos-service.service`).
```
Comment (makkes) [inline pages/1.12/monitoring/index.md]: That should be `dcos-diagnostics.service`.

**Comment by janisz** [comment]
https://jira.mesosphere.com/browse/DCOS_OSS-5146

**Comment by janisz** [comment]
@pavisandhu Can you take a look?

File: pages/1.12/monitoring/index.md [hunk 2]
```diff
@@ -92,9 +92,13 @@ You can sort system health by `systemd` unit. However, this search can bring up
...
 
+### Diagnostics bundle contents
+
+The contents of the generated bundle are not stable over time and any internal or third party bundle analysis tooling should be programmed very defensively in this regard.
+
 ## Troubleshooting
 
-If you have any problems, you can check if the diagnostics service is running by SSHing to the Mesos leading master and checking the `systemd` status of the diagnostics component (`dcos-d3t.service`).
+If you have any problems, you can check if the diagnostics service is running by SSHing to the Mesos leading master and checking the `systemd` status of the diagnostics component (`dcos-diagnostics.service`).
```
Comment (ck4adventure) [inline pages/1.12/monitoring/index.md]: @makkes @jdef @janisz Does this/these edits need to be backported to 1.10 and 1.11? Specifically the change to `dcos-diagnostics.service`????

**Pull request merged**